test/unit/operations/abstract_command.test.ts

```diff
@@ -0,0 +1,58 @@
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+
+import {
+  type Callback,
+  type ClientSession,
+  CommandOperation,
+  type CommandOperationOptions,
+  type Document,
+  type OperationParent,
+  Server,
+  ServerDescription
+} from '../../mongodb';
+import { topologyWithPlaceholderClient } from '../../tools/utils';
+
+class ConcreteCommand<T> extends CommandOperation<T> {
+  constructor(parent?: OperationParent, options?: CommandOperationOptions) {
+    super(parent, options);
+  }
+
+  async executeCommand(
+    server: Server,
+    session: ClientSession | undefined,
+    cmd: Document
+  ): Promise<any> {
+    return super.executeCommand(server, session, cmd);
+  }
+
+  protected executeCallback(
+    server: Server,
+    session: ClientSession | undefined,
+    callback: Callback<any>
+  ) {
+    return [server, session, callback];
```

Comment:
To just simulate any random return value from the server this can be a hardcoded `{ok: 1}`